driver-legacy/src/main/com/mongodb/MongoClient.java

```diff
@@ -83,7 +107,18 @@
  * @since 2.10.0
  */
 @SuppressWarnings("deprecation")
-public class MongoClient extends Mongo implements Closeable {
+public class MongoClient implements Closeable {
```

Comment:
This is the most complicated change since the Mongo base class is deleted and so all of its functionality had to be merged into this class